plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManager.kt

```diff
@@ -148,11 +148,17 @@ internal class LocationPuckManager(
     locationLayerRenderer.hide()
   }
 
+  /**
+   * function to set scaling for [LocationPuck].
+   * In order to keep 3D puck size constant across all zoom levels, we interpolate the model based on
+   * current zoom level. MIN_ZOOM, MAX_ZOOM are used as two anchor points to calculate
+   * the scale expression.
+   */
```

Comment:
please also update the doc of `LocationPuck3D#modelScale`

Reply:
will create another pr to update the doc.